core/src/main/scala/kafka/log/LogLoader.scala

```diff
@@ -78,7 +78,8 @@ class LogLoader(
   recoveryPointCheckpoint: Long,
   leaderEpochCache: Option[LeaderEpochFileCache],
   producerStateManager: ProducerStateManager,
-  numRemainingSegments: ConcurrentMap[String, Int] = new ConcurrentHashMap[String, Int]
+  numRemainingSegments: ConcurrentMap[String, Int] = new ConcurrentHashMap[String, Int],
+  isRemoteLogEnabled: Boolean = false,
```

Comment:
Could we add the new param to the javadoc?

Reply:
Opened a PR to address it: https://github.com/apache/kafka/pull/15179. PTAL